hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsConfigKeys.java

```diff
@@ -193,6 +193,8 @@ public final class HddsConfigKeys {
       "hdds.x509.renew.grace.duration";
 
   public static final String HDDS_X509_RENEW_GRACE_DURATION_DEFAULT = "P28D";
+  public static final String HDDS_NEW_KEY_CERT_DIR_NAME_SUFFIX = "-new";
```

Comment:
I think we should make this conform with what we have in hadoop for previous-current, and based on the pattern of previous I would call the new as next, so I believe we might have these constants defined as "-next" and "-previous".

Reply:
Sure.